app/views/users/edit.html.erb

```diff
@@ -1,2 +1,24 @@
-<h1>Users#edit</h1>
-<p>Find me in app/views/users/edit.html.erb</p>
+<div class="containter">
+	<div class="row">
+		<div class="col-sm-6 col-sm-offset-3 user_signup_form">
+			<%= form_for @user, url: users_path, html: { multipart: true } do |f| %>
```

Comment:
@cameronjacoby Tomorrow I will show Dave the new url, since I don't use user_path ti link to user show page. Cuz we have ta and student, so I make ta_path and student_path :smile: